src/components/UI/Select.vue

```diff
@@ -37,3 +47,36 @@ export default defineComponent({
   }
 })
 </script>
+
+<style lang="scss" module>
+.display {
```

Comment:
```suggestion
.select {
```
こっちがselectにするほうがいいかな